users/vieting/experiments/switchboard/ctc/feat/baseline_args.py

```diff
@@ -197,6 +205,8 @@ def get_returnn_config(
 
     if audio_perturbation:
         prolog += get_code_for_perturbation()
+        if recognition and "pre_process" in datasets["dev"]["audio"]:
+            del datasets["dev"]["audio"]["pre_process"]
```

Comment:
It is not used, but it leads to errors during compile time, because the code is not found in the config. That's why the function call has to be removed for recognition